atom/renderer/lib/web-view/web-view.js

```diff
@@ -430,6 +432,11 @@ var registerWebViewElement = function() {
     proto[m] = createNonBlockHandler(m);
   }
 
+  for (k=0, len2 = webFrameMethods.length; k < len2; k++) {
```

Comment:
No need to follow the CoffeScript generated code, use `for let method of webFrameMethods` instead.